Gopkg.toml

```diff
@@ -180,3 +180,8 @@ required = [
 [[override]]
   name = "github.com/golang/glog"
   source = "github.com/istio/glog"
+
+[[constraint]]
```

Comment:
this isn't needed, just use it somewhere and make the vendor PR from
`make depend.update`